sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -2194,7 +2194,7 @@ class DataSourceV2SQLSuite
     withTempView("v") {
       sql("create global temp view v as select 1")
       val e = intercept[AnalysisException](sql("COMMENT ON TABLE global_temp.v IS NULL"))
```

Comment:
What happened if we apply this command to DSV1 tables? How many DDLs/DMLs are applied only to DSV2 tables?

Reply:
I think all commands support v1 table: they either have a v1 command, or the v2 command works out of the box thanks to v2 session catalog. We need to add test though.